src/Microsoft.AspNetCore.Mvc.Core/Internal/MiddlewareFilterBuilderStartupFilter.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Mvc.Internal
+{
+    internal class MiddlewareFilterBuilderStartupFilter : IStartupFilter
```

Comment:
Should this be pub internal or true internal?

Reply:
real internal 

the `.Internal` namespace isn't needed.

Reply:
It is .Internal because it is in the same directory as the MiddlewareFilterBuilder.